infrastructure/repository/bill.py

```diff
@@ -32,27 +37,58 @@ def listCompletedBillFromAndTo(self,f:datetime,t:datetime) -> List[BillDTO]:
             return   [BillDTO.from_orm(bill) for bill in bills]
         except SQLAlchemyError as e:
             raise SQLALCHEMY_ERROR(e)
-
+        
+    def listSmallCompletedBillFromAndTo(self,f:datetime,t:datetime) -> List[BillSmallDTO]:
```

Comment:
When you add new methods/function in repository. You need to add these in core/protocols too. Which is like somewhat abstracts of the repository.